rex-via-ansible.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+h 52-rex-date.log "job-invocation create --inputs \"command='date'\" --job-template 'Run Command - Ansible Default' --search-query 'name ~ container'"
```

Comment:
```suggestion
h 52-rex-date-ansible.log "job-invocation create --inputs \"command='date'\" --job-template 'Run Command - Ansible Default' --search-query 'name ~ container'"
```
We are using this log name to identify command - this way result would get mixed with the other command.

Also please put this command directly into `experiment/full.sh` (please pull to make sure you have latest git)